iModelCore/ECDb/Tests/NonPublished/InstanceReaderTest.cpp

```diff
@@ -683,12 +683,13 @@ TEST_F(InstanceReaderFixture, check_option_DO_NOT_TRUNCATE_BLOB) {
                 "y": 9.73592815626386,
                 "z": 9.735928156263856
             },
-            "geometryStream": "encoding=base64;ywCAAjAABgAA+AAAAAEAAAAIDQgBAUAEAAAAMAAAABwAAAAYABQADAUeEQEIBgAHBRgBAQwBAQDwASQJAUALAAAAqAAAAGJnMDAwMWZiEAUXEAoADgAHBUIACgUQCAAHDAUIyAYAfAAEAAYAAAC8t0aTy3gjQNTy0dk2l6Q8BOGMD2d0zbxZPdLR+8bSvLS6W8O77KW8vQ0oBT8IANg8CQgg0LyQPKeSAhKeERAEPLoyKAAk4LwYLURU+yH5vwkIJAlAAQAAAAAAAAA="
```

Comment:
Does this change also affect the test BRep data used on the typescript side?

[brepdata1.json](https://github.com/iTwin/itwinjs-core/blob/master/core/backend/src/test/assets/brepdata1.json)

Reply:
No the change here is that GeomStream is now returned as decompressed blob. So you seen that change in the test. Previously it was returned as snappy compressed blob. It could not be used on itwin.js side anyway as there is no way to uncompress it. 

We use instance reader to get the decompress geometry stream then send it from itwin.js side back to native to convert it into GeometryStreamProps (JSON).